.github/workflows/check.yml

```
@@ -22,16 +22,23 @@ jobs:
           packages: libxtst-dev libevdev-dev libxdo-dev
           version: 1.0
 
+      - name: Ensure each file respect the max number of lines
+        run: |
+          find . -iname "*.toml" -exec awk 'FNR == 100001 { print "Error: " FILENAME " exceeds 100,000 lines, consider to split it!"; err=1; nextfile } END { exit err }' {} +
+
       - name: Dependencies
         run: |
           sudo apt-get install libxtst-dev libevdev-dev libxdo-dev --assume-yes
+      
       - uses: actions-rs/toolchain@v1
```

actionlint (1.7.12)

[error] the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)